### PR TITLE
Fix for Json parsing

### PR DESCRIPTION
**Pull request by petarjuki7**
## CLI JsonRpc

Quick fix for json parsing

### CODE REVIEW

**Comment by frdomovic** [comment]
@petarjuki7 Please leave the commands in comment so I can test it fast

**Comment by petarjuki7** [comment]
cargo run -p meroctl -- --node-name node2 json-rpc --call-type mutate --context-id <context_id> --method create_post --args-json '{title: naslov, content: sadrzaj}'

**Comment by frdomovic** [comment]
Screenshot 2024-10-07 at 11 02 04

Not sure how to fix this. Any idea?

**Comment by frdomovic** [comment]
It doesn't take the right path. Should be  /Users/frandomovic/Desktop/job/core/data/node1 . 

Screenshot 2024-10-07 at 11 27 01

**Comment by petarjuki7** [comment]
Tested different commands, everything works:

`cargo run -p meroctl -- --node-name node2 json-rpc --call-type mutate --context-id 4kmzhXNTqJcUrScCZqYvgME7LVBtaCMdPzqGQtubzT5s --method create_post --args-json '{"title": "Ola", "content": "bola"}'`

`cargo run -p meroctl -- --node-name node2 json-rpc --call-type query --context-id 4kmzhXNTqJcUrScCZqYvgME7LVBtaCMdPzqGQtubzT5s --method posts`

`cargo run -p meroctl -- --node-name node2 json-rpc --call-type query --context-id 4kmzhXNTqJcUrScCZqYvgME7LVBtaCMdPzqGQtubzT5s --method post --args-json '{"id": 4}'`

**Review by frdomovic — APPROVED** [review]

File: crates/meroctl/src/cli/jsonrpc.rs [hunk 1]
```diff
@@ -58,11 +58,13 @@ impl JsonRpcCommand {
...
         let url = multiaddr_to_url(multiaddr, "jsonrpc/dev")?;
 
+        let json_payload: Value = serde_json::from_str(&self.args_json)?;
+
         let payload = match self.call_type {
             CallType::Query => RequestPayload::Query(QueryRequest::new(
                 self.context_id,
                 self.method,
-                json!(self.args_json),
+                json_payload,
```
Comment (chefsale) [inline crates/meroctl/src/cli/jsonrpc.rs:67]: maybe just: `serde_json::from_str(&self.args_json)?`

**Pull request merged**